data/mods/Aftershock/maps/mapgen/arcology_ground.json

```diff
@@ -0,0 +1,835 @@
+        "________________óó,,,,,,",
+        "_______2________||,,,,,,",
+        "________________||,,,,,,",
+        "3_______________((,,,,,,",
+        "________________((,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "||||||||||||((||||,,,,,,",
+        ",,,,,,,,,,,,,,,,,,,,,,,,",
+        ",,,,,,,,,,,,,,,,,,,,,,,,",
+        ",,,,,,,,,,,,,,,,,,,,,,,,"
+      ],
+      "palettes": [ "arcology_exterior" ],
+      "nested": {
+        "1": { "chunks": [ [ "arcology_8x8_electronic", 20 ], [ "arcology_8x8_foodplace", 20 ] ] },
+        "2": { "chunks": [ [ "arcology_9x9_candy_shop", 20 ], [ "city_block_9x9_beauty_shop", 20 ], [ "arcology_9x9_vitamin_shop", 20 ] ] },
+        "3": { "chunks": [ [ "lab_spawn_7x7", 20 ], [ "lab_spawn_7x7_crossdoors", 20 ] ] }
+      },
+      "place_loot": [ { "group": "TBD", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 50, "repeat": [ 1, 4 ] } ],
+      "place_monsters": [ { "monster": "AFS_GROUP_ARCOLOGY_EXTERIOR", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 10, "density": 0.1 } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "arcology_exterior_e_side_robots", "arcology_exterior_e_side_uplifts" ],
+    "weight": 100,
+    "object": {
+      "rows": [
+        "____1___________óó,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________((,,,,,,",
+        "________________((,,,,,,",
+        "________________((,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "____1___________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________((,,,,,,",
+        "________________((,,,,,,",
+        "________________((,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________||,,,,,,",
+        "________________óó,,,,,,"
+      ],
+      "palettes": [ "arcology_exterior" ],
+      "nested": { "1": { "chunks": [ [ "workoutroom_12x12", 20 ], [ "tvroom_12x12", 20 ] ] } },
+      "place_loot": [ { "group": "TBD", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 50, "repeat": [ 1, 4 ] } ],
+      "place_monsters": [ { "monster": "AFS_GROUP_ARCOLOGY_EXTERIOR", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 10, "density": 0.1 } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "arcology_exterior_e_side_entrance_robots", "arcology_exterior_e_side_entrance_uplifts" ],
+    "weight": 100,
+    "object": {
+      "rows": [
+        "________________óó,,,,,,",
+        "_________1______||,,,,,,",
```

Comment:
Error here.
12:58:10.277 ERROR : src/main_menu.cpp:863 [bool main_menu::new_character_tab()] Error: Json error: data/mods/Aftershock/maps/mapgen/arcology_ground.json:351:19: format: rows: row 2 column 10: '1' is not in 'terrain', and no 'fill_ter' is set!

      "rows": [
        "________________óó,,,,,,",
        "_________1
                  ^
                   ______||,,,,,,",
        "________________||,,,,,,",
        "________________||,,,,,,",

12:58:42.065 : Log shutdown.